### ISSUE TITLE
WPRig v2.0 and WPML

### ISSUE THREAD

**Issue by idea--list** [wprig/wprig#516]
## Issue Overview

Just generated a theme with WPRig v2.0 without any modification on a production site that uses has a bunch of active plugins, including WPML. Was surprised as the only thing that has been loaded was the site title and site description, nothing else got rendered.
So i began the usual cycle searching the plugin that might cause that and ultimately it turns out if WPML is active then the theme generated with WPRig v2.0 just freezes before loading the actual content of the page.
Unluckily there is nothing logged in the console, so probably it seems rather like a php conflict

## Steps to Reproduce

1. Generate a theme with WPRig v2.0
2. Activate it on a site that uses WPML

## Expected Behavior

The theme should not break with a popular plugin like WPML

**Comment by benoitchantre** [comment]
Yes: replace WPML by Polylang (free or Pro). It will be much faster and easier to use.

**Comment by idea--list** [comment]
@benoitchantre 
No, i just will not replace WPML to polylang because i had that before and it was crap.

**Comment by benoitchantre** [comment]
Cc. @felixarntz for feeback on that.

**Comment by mor10** [comment]
bump @felixarntz
